gcloud/logging/logging_service_v2_api.py

```diff
@@ -136,73 +178,81 @@ def write_log_entries(
             labels = []
         if entries is None:
             entries = []
-        write_log_entries_request = logging_pb2.WriteLogEntriesRequest(
-            log_name=log_name,
-            resource=resource,
-            labels=labels,
-            entries=entries,
-            **kwargs)
-        return self.write_log_entries_callable()(write_log_entries_request)
-
-    def write_log_entries_callable(
-            self,
-            is_retrying=None,
-            max_attempts=None):
-        return api_callable.idempotent_callable(
+        req = logging_pb2.WriteLogEntriesRequest(log_name=log_name,
+                                                 resource=resource,
+                                                 labels=labels,
+                                                 entries=entries)
+        settings = self._defaults['write_log_entries'].merge(options)
+        return api_callable.ApiCallable(
             self.stub.WriteLogEntries,
-            is_retrying=is_retrying,
-            max_attempts=max_attempts,
-            defaults=self.defaults)
-
-    def list_log_entries(
-            self,
-            project_ids=None,
-            filter_='',
-            order_by='',
-            **kwargs):
+            settings=settings)(req,
+                               metadata=self._headers)
+
+    def list_log_entries(self,
+                         project_ids=None,
+                         filter_='',
+                         order_by='',
+                         options=None):
         """
-        Lists log entries.  Use this method to retrieve log entries from Cloud
-        Logging.  For ways to export log entries, see
-        [Exporting Logs](/logging/docs/export).
+        Lists log entries.
+        Use this method to examine log entries from Cloud Logging.
+        See `Exporting Logs <https://cloud.google.com/logging/docs/export>`_
+        log entries out of Cloud Logging.
+
+        :type project_ids: list of string
+        :type filter: string
```

Comment:
Here's where things get confusing. What can I put in here? What does a valid filter string look like? How do I do multiple filter strings?